### PR TITLE
Revamp documentation

### PR DESCRIPTION
**Pull request by Joonpark13**
Various changes to documentation - adding more info as well as removing outdated ones. Unit testing philosophy changes coming in a later ticket.

### CODE REVIEW

File: docs/ONBOARDING.md [hunk 1]
```diff
@@ -71,131 +71,10 @@ You probably don't need to master these concepts before starting to contribute t
...
+3. At this point, if you feel a bit shaky or still confused, you should touch base with the student lead/an experienced member and go through a simple example React app together. If you're ready to move on...
 
 4. You should make sure you are familiar with git in some way. If you use a github GUI, that's fine - or maybe you've done basic git on the command line before. If you're unfamiliar with git, I *highly* recommend that you go through this [git tutorial](https://www.codecademy.com/learn/learn-git).
 
-5. If you feel good about all that, you can move on to the practice ticket below.
+5. If you feel good about all that, you can move on [PRACTICE_TICKET.md](PRACTICE_TICKET.md).
 
 6. At some point later down the line, you should come back and do the [Redux tutorial](https://redux.js.org/basics/basic-tutorial). Redux is our state management solution and it's a core part of how our app functions. It's a bit difficult of a concept to grasp at first, but you should try to understand it after you've gotten a bit of a look into the codebase.
-
-## Practice Ticket
```
Comment (Joonpark13) [inline docs/ONBOARDING.md:82]: Moved this whole section out into a separate file

File: docs/REVIEWING.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+## Take your time
+
+There's a huge difference between just mindlessly skimming through someone's code and actually sitting down to understand it. The latter will take much more time, and you will definitely have to read through the code multiple times. That's expected. It's not always easy to discern someone else's intent by reading their code. You should feel free to reach out to them on slack and ask questions. You reviewing their PR isn't always for you to critique their work - sometimes it's an opportunity for you to learn something from what they wrote. It goes both ways.
+
+## A second pair of eyes
+
+Don't hesitate to leave a comment about anything, even if you think it might be something obvious. You would be surprised how different a second pair of eyes sees the same pice of code. Offering critique is appreciated, not frowned upon - everyone's here to learn. If you suggest improvements to their code, do so in a constructive manner, but don't let it slide by just because "it's a small thing." Small things add up very quickly to tech debt that can later become unmanageable.
```
Comment (kevinjihwanlee) [inline docs/REVIEWING.md]: small typo, should be: You would be surprised how different a second pair of eyes sees the same **piece** of code

File: docs/REVIEWING.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+## Take a step back
+
+If you feel like you've gone through all the files and left all the comments you wanted, take a step back and once again see if you understand the goal of the PR. Now that you've gone through the majority of the code, you may understand the PR in a way you didn't before, and might have more wholistic comments that you might want to say.
```
Comment (kevinjihwanlee) [inline docs/REVIEWING.md]: small typo, should be:  Now that you've gone through the majority of the code, you may understand the PR in a way you didn't before, and might have more **holistic** comments that you might want to say.

**Review by kevinjihwanlee — APPROVED** [review]

File: docs/REVIEWING.md (hunk 2 above)
Comment (Joonpark13) [reply docs/REVIEWING.md]: The value of the second pair of eyes coming in handy on the passage about a second pair of eyes - so meta

**Pull request merged**